mne/decoding/time_gen.py

```diff
@@ -139,11 +143,11 @@ class GeneralizationAcrossTime(object):
 
     .. versionadded:: 0.9.0
     """  # noqa
-    def __init__(self, cv=5, clf=None, train_times=None,
+    def __init__(self, cv=5, clf=None, train_times=None, predict_shape=1,
                  predict_mode='cross-validation', n_jobs=1):
 
         from sklearn.preprocessing import StandardScaler
-        from sklearn.svm import SVC
+        from sklearn.linear_model import SGDClassifier
```

Comment:
SVC is gets too heavy on memory when the data is not decimated because it keeps support_vectors_ (n_support * n_features) on every time point.

The alternatives are LinearSVC and SGDClassifier which are lighter. However, LinearSVC was sig. slower than SVC(kernel='linear') (I'm not sure why, but apparently it may be due to dense versus sparse matrices expectation?). SGDClassifier seemed faster.

What would you recommand